### PR TITLE
[now-cli] Enhance code-formatting

### PR DESCRIPTION
**Pull request by mstruebing**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,6 +22,12 @@
     "buffer-replace": "1.0.0",
     "eslint": "6.2.2",
     "eslint-config-prettier": "6.1.0",
+    "eslint-config-react-app": "5.0.1",
+    "eslint-config-synacor": "3.0.4",
+    "eslint-plugin-ember": "7.0.0",
+    "eslint-plugin-node": "9.2.0",
+    "eslint-plugin-react-app": "6.0.1",
+    "eslint-plugin-vue": "5.2.3",
```
Comment (lucleray) [inline package.json]: These dependencies are not necessary to run the project. Maybe they were required by your global eslint config?

Comment (mstruebing) [reply package.json]: This is currently in because `eslint` needed it when running the pre-commit-hook. I will investigate it furhter later.

Comment (mstruebing) [reply package.json]: Removed them right now, lets see if `circleci` runs through.

**Comment by mstruebing** [comment]
Can someone help me out why the tests are failing?

**Comment by TooTallNate** [comment]
Great work @mstruebing! However, I think we should avoid formatting changes to the test fixtures, because most of them are auto-generated from the projects/frameworks that they are testing.

**Comment by mstruebing** [comment]
Okay, I will remove the formatting of these - maybe this will already fix the tests.

**Comment by mstruebing** [comment]
Can someone restart the failing tests please?

**Comment by mstruebing** [comment]
After rebase tests are green :)

File: .ecrc [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  "Exclude": ["\/fixtures\/"],
+  "AllowedContentTypes": [],
+  "PassedFiles": [],
+  "Disable": {
+    "EndOfLine": false,
+    "Indentation": false,
+    "InsertFinalNewline": false,
+    "TrimTrailingWhitespace": false
+  }
+}
```
Comment (styfle) [inline .ecrc:16]: What is ecrc?

Comment (mstruebing) [reply .ecrc:16]: Configuration file for editorconfig-checker: https://editorconfig-checker.github.io/

Comment (styfle) [reply .ecrc:16]: We already have eslint and prettier. I don't think we want another formatting config. cc @lucleray

Comment (lucleray) [reply .ecrc:16]: I think prettier/eslint should be enough. We have `.editorconfig` as a "nice to have" but prettier/eslint are the "authority" for formatting/linting.

Comment (mstruebing) [reply .ecrc:16]: Okay.

**Pull request closed without merging**